spec/models/reservation_spec.rb

```diff
@@ -87,705 +73,830 @@
     end
   end
 
-  describe '.number_for' do
-    before(:each) do
-      @source = FactoryGirl.build_pair(:valid_reservation)
-    end
+  describe '#number_for' do
```

Comment:
Aren't these class methods? I thought the convention was `Class.class_method` and `Class#instance_method`. Same for the next bunch of tests.